src/rainfly/static/examples/bypass/main.js

```diff
@@ -0,0 +1,9 @@
+// Use `context` as the AudioContext
+// @sampleRate = 48000
```

Comment:
Is this a parameter?  Generally the output device sample rate is used if nothing is specified (https://webaudio.github.io/web-audio-api/#dictionary-audiocontextoptions-members).

Reply:
Yes this is a parameter that we preprocess. If not specified, default is also 48000.

Reply:
Would the Canopy parameter syntax of `@sampleRate 48000` be preferred?

Reply:
Either syntax looks fine to me.  Is there a place the parameters are documented?

Reply:
This is the only parameter that we have.

Reply:
I feel like this should be documented somewhere, either a readme or in this repository's wiki.  If there's not a natural place to do so in this PR that's totally fine; I can file an issue to follow-up with adding documentation (something like "Document process for adding new Rainfly examples" and note that the samplerate syntax should be included in the documentation).  What do you think?

Reply:
Also, just to make sure I'm not misunderstanding, the reason I asked this question is it looks like a comment.  So we are processing the comments and extracting parameters from them, correct?

Reply:
I totally agree with you that there should be a guide for writing/reading examples. I was also thinking that we could add the user guide under the `Help` dropdown if not a dedicated README/guide. If you could open up an issue for this, that would be great! (It'd be easier to address it in a separate PR!)

Correct regarding your second comment. We are parsing params via `// @paramName = <value>` syntax. Maybe the example could have a comment like `// parameters: ` to start.

Reply:
Done with #404